src/wp-admin/includes/update-core.php

```diff
@@ -1637,14 +1637,16 @@ function _upgrade_440_force_deactivate_incompatible_plugins() {
 /**
  * @access private
  * @ignore
- * @since 5.9.0
+ * @since 5.8.0 The minimum compatible version of Gutenberg is 10.7.
+ * @since 5.9.0 The minimum compatible version of Gutenberg is 11.9.
+ * @since 6.1.1 The minimum compatible version of Gutenberg is 14.1.
  */
-function _upgrade_590_force_deactivate_incompatible_plugins() {
-	if ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '11.9', '<' ) ) {
+function _wp_upgrade_force_deactivate_incompatible_plugins() {
```

Comment:
@desrosj What do you think? 

* Added the `@since` for each release.
* Renamed the function to no longer be named by its release number.
* But added a `wp` prefix to avoid any naming conflicts outside of Core.

Reply:
Hmm, or maybe the function name should be:

```
_upgrade_core_force_deactivate_incompatible_plugins()
```

as it better aligns with the naming conventions in the same file. Thoughts?

Reply:
I went ahead and renamed in https://github.com/WordPress/wordpress-develop/pull/3559/commits/3588cef15aea3a36c4fc0dbb9b175dda3bae0f4c. The function name aligns better with the functions in the same file.